bin/console

```diff
@@ -6,6 +6,7 @@ require 'record_store'
 
 RecordStore.zones_path = File.expand_path('../../dev/zones', __FILE__)
 RecordStore.config_path = File.expand_path('../../dev/config.yml', __FILE__)
+RecordStore.zones_path = File.expand_path('../../dev/templates', __FILE__)
```

Comment:
```suggestion
RecordStore.templates_path = File.expand_path('../../dev/templates', __FILE__)
```

Reply:
Do we ignore the `zones_path` and `templates_path` parameters in the `config.yml` when using the console ? (I guess yes since it's just for testing ?)

Reply:
whoops thanks for the save - i think i just miscommitted what i have locally. let me repush